extension.js

```diff
@@ -97,11 +97,11 @@ function createGetterAndSetter(textPorperties)
 
             let code = 
 `
-\tpublic ${type} get${Attribute}() {
+\tpublic ${type} ${type.startsWith('bool') ? 'is' : 'get'}${Attribute}() {
```

Comment:
what about if the type value is `Boolean` or `boolean` like your issue examples? Is the code case insensitive?

Reply:
@afmicc 

JavaScript `String.startsWith` method is case sensitive.

---

About the results of the added code:

- getter of `Boolean` (first letter uppercase) will be named as `getProperty()`, since it can also be null.

- getter of `boolean` (all lowercase) will be named as `isProperty()`, since it will be either `true` or `false`.

Syntax examples:

- 1) `if (test.isProperty()) // ...` can be used with `boolean` types, but not with `Boolean` types.

- 2) `if (test.getProperty() == true)` // ... is redundant to `boolean` types (some editors will warn about it). However it is the only way to check if a `Boolean` type is true.

Reply:
Nice! greate explanation